packages/core-data/src/entity-types/post-type.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Internal dependencies
+ */
+import type { Context, OmitNevers } from './helpers';
+import type { BaseEntityRecords as _BaseEntityRecords } from './base-entity-records';
+
+declare module './base-entity-records' {
+	export namespace BaseEntityRecords {
+		/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+		export interface PostType< C extends Context > {
```

Comment:
The description of these fields is copied from core [REST API schema](https://github.com/WordPress/WordPress/blob/ad2f51c90a16e3e7b9cd14c855c17c2321d2611a/wp-includes/rest-api/endpoints/class-wp-rest-post-types-controller.php#L320-L326).